tsdoc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
+  "tagDefinitions": [
+    {
+      "tagName": "@hideconstructor",
+      "syntaxKind": "modifier"
```

Comment:
To resolve `tsdoc-undefined-tag: The TSDoc tag "@hideconstructor" is not defined in this configuration` error, I referenced this link https://github.com/microsoft/tsdoc/issues/213